.github/workflows/deploy.yml

```
@@ -0,0 +1,51 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)